recipes/libmaxminddb/all/conanfile.py

```diff
@@ -81,7 +80,6 @@ def package_info(self):
 
         if self.settings.os != "Windows":
```

Comment:
Should this line include checking if `self.options.get_safe("with_binaries")` is true?

Reply:
@RubenRBS 
Thank you for your comment.
I fixed it.